### ISSUE TITLE
Not able to setup the environment to run the code

### ISSUE THREAD

**Issue by Dhandeep10** [yoheinakajima/instagraph#32]
After completing all the steps in README file, and troubleshooting some things by my self. I got the following error while running **"python main.py"** in terminal
```
NotOpenSSLWarning: urllib3 v2.0 only supports OpenSSL 1.1.1+, currently the 'ssl' module is compiled with 'LibreSSL 2.8.3'. See: https://github.com/urllib3/urllib3/issues/3020
  warnings.warn(
Traceback (most recent call last):
  File "/Users/dhandeepsingh/Desktop/instagraph/main.py", line 5, in <module>
    from bs4 import BeautifulSoup
ModuleNotFoundError: No module named 'bs4'
```

How to fix this issue??

**Comment by admalekar** [comment]
Seems like the beautifulsoup4 package was not installed. You can pip install the requirements again and check if beautifulsoup4 is installed.

It would be better if we create a virtualenv for the dependencies.

@yoheinakajima I can update the readme to add steps for creating a virtualenv if you would like.

**Comment by Dhandeep10** [comment]
I tried installing, but it said that it is already installed.
So any other help?

**Comment by Tayyab-R** [comment]
> Seems like the beautifulsoup4 package was not installed. You can pip install the requirements again and check if beautifulsoup4 is installed.
> 
> It would be better if we create a virtualenv for the dependencies.
> 
> @yoheinakajima I can update the readme to add steps for creating a virtualenv if you would like.

yes a virtual env would be better. Give it a try hope it will solve the issue.

**Comment by admalekar** [comment]
@Dhandeep10 Try this steps if you are on windows:

git clone https://github.com/yoheinakajima/instagraph.git
cd instagraph
python -m venv venv
.\venv\Scripts\activate
pip install -r requirements.txt
python main.py
